### PR TITLE
OHRM5X- 272 : Education screen test scripts

### PR DESCRIPTION
**Pull request by ShenaliFernando**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (samanthajayasinghe) [inline .vscode/launch.json:1]: don't commit those files ..pls

File: symfony/test/functional/cypress.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "http://php80/orangehrm/symfony/web/index.php",
+    "baseUrl": "https://os50al9-os-guardian.orangehrm.com/web/index.php",
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress.json:2]: don't commit this file too..

File: symfony/test/functional/package.json [hunk 3]
```diff
@@ -1,5 +1,8 @@
 {
   "devDependencies": {
-    "cypress": "^7.3.0"
+    "cypress": "^7.3.0",
+    "cypress-promise" : "^1.1.0"
   }
+
```
Comment (samanthajayasinghe) [inline symfony/test/functional/package.json:6]: unwanted lines

**Review by samanthajayasinghe — CHANGES_REQUESTED** [review]

File: symfony/test/functional/package.json (hunk 3 above)
Comment (ShenaliFernando) [reply symfony/test/functional/package.json:6]: Removed extra lines

File: .vscode/launch.json (hunk 1 above)
Comment (ShenaliFernando) [reply .vscode/launch.json]: launch.json file removed

File: .gitignore [hunk 4]
```diff
@@ -11,3 +12,10 @@ logInsatall.log
 symfony/test/functional/node_modules
 symfony/tests/functional/yarn.lock
 coverage/
+.vscode
```
Comment (RajithaKumara) [inline .gitignore:15]: remove this line since rule already added 2nd line

File: .gitignore [hunk 5]
```diff
@@ -11,3 +12,10 @@ logInsatall.log
...
+
+symfony/test/functional/node_modules
+symfony/test/functional/*.log
+symfony/test/functional/cypress/screenshots
+symfony/test/functional/cypress.json
+symfony/test/functional/cypress/fixtures/user.json
```
Comment (RajithaKumara) [inline .gitignore:21]: add new line

File: symfony/test/functional/cypress.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "http://php80/orangehrm/web/index.php",
+    "baseUrl": "http://php80/orangehrm/symfony/web/index.php",
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress.json:2]: revert change

File: symfony/test/functional/cypress/fixtures/admin.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+    "admin": {
+        "userName": "admin",
+        "password": "admin@OHRM123",
+        "fullName": "Shenali Fernando"
+    } 
+}
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress/fixtures/admin.json:7]: add new line

File: symfony/test/functional/cypress/integration/orangehrm/admin/education.spec.js [hunk 8]
```diff
@@ -0,0 +1,143 @@
+import user from '../../../fixtures/admin.json'
+import charLength from '../../../fixtures/charLength.json'
+import promisify from 'cypress-promise'
+
+    //View Education List Page
+    it('Check education view page', () => {
```
Comment (RajithaKumara) [inline symfony/test/functional/cypress/integration/orangehrm/admin/education.spec.js:6]: wrap tests with one describe

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (ShenaliFernando) [reply .gitignore:15]: Removed .vscode in line 15

File: .gitignore (hunk 5 above)
Comment (ShenaliFernando) [reply .gitignore:21]: New line added.

File: symfony/test/functional/cypress/fixtures/admin.json (hunk 7 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/fixtures/admin.json:7]: New line added

File: symfony/test/functional/cypress/integration/orangehrm/admin/education.spec.js (hunk 8 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/integration/orangehrm/admin/education.spec.js:6]: Done

File: .gitignore [hunk 9]
```diff
@@ -11,3 +12,9 @@ logInsatall.log
 symfony/test/functional/node_modules
 symfony/tests/functional/yarn.lock
 coverage/
+
+symfony/test/functional/node_modules
+symfony/test/functional/*.log
+symfony/test/functional/cypress/screenshots
+symfony/test/functional/cypress.json
+symfony/test/functional/cypress/fixtures/user.json
```
Comment (samanthajayasinghe) [inline .gitignore:20]: I think we should git ignore admin.json, not the user.json

File: symfony/test/functional/cypress/fixtures/admin.json [hunk 10]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (samanthajayasinghe) [inline symfony/test/functional/cypress/fixtures/admin.json:1]: This file shouldn't be here

File: .gitignore (hunk 9 above)
Comment (ShenaliFernando) [reply .gitignore]: Added admin.json to git ignore

File: symfony/test/functional/cypress.json (hunk 6 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress.json]: Done

File: symfony/test/functional/cypress/fixtures/admin.json (hunk 10 above)
Comment (ShenaliFernando) [reply symfony/test/functional/cypress/fixtures/admin.json:1]: Admin.json added to the git ignore file

File: symfony/test/functional/.gitignore [hunk 11]
```diff
@@ -1,4 +1,5 @@
-/node_modules
-/cypress.json
-/cypress/videos
-/cypress/screenshots
+node_modules
+*.log
+cypress/screenshots
+cypress/fixtures/user.json
```
Comment (samanthajayasinghe) [inline symfony/test/functional/.gitignore:4]: Why user.json in .gitignore ?

Comment (ShenaliFernando) [reply symfony/test/functional/.gitignore]: Removed user.json from .gitignore.

**Pull request merged**